### ISSUE TITLE
Reproduction of Experiments

### ISSUE THREAD

**Issue by atakansite** [phquang/Continual-Normalization#4]
Hello, 
We are trying to reproduce some of experiments in the Continual Normalization: Rethinking Batch Normalization For Online Continual Learning paper. We will have some questions about the paper and repository.
 
Are there any exact list of hyperparameters of the experiments? We couldn't reproduce published results. In addition, how can we achieve datasets and model settings for Long-Tailed Online Continual Learning experiments which made in Section 5.3. NUS-WIDEseq and COCOseq benchmarks , model and hyperparameters are not available in source code and mammoth framework.

Best,

Atakan Site, Mehmet Selahaddin Şentop

**Comment by phquang** [comment]
Hi,
Thanks for your interests in our work. We used ER and DER++ as implemented in the ``mammoth`` folder (forked from this repo: https://github.com/aimagelab/mammoth/). Hyper-parameters configurations (learning rates and other hyper-parameters)  can be found at ``/mammoth/utils/best_args.py``. The hyper-parameter of our CN is the number of groups, which you can set to be 8 or 32 as described in the README file, which should produce the results in Table 4, Section 5.2.

For Section 5.3, you can clone the PRS repo (https://github.com/cdjkim/PRS), follow our provided script to replace BN with CN, and run the experiments as instructed in the PRS repo. The number of groups in this experiment should be 4.
